database/seeders/LaratrustSeeder.php

```diff
@@ -47,7 +47,7 @@ public function run() {
         ]);
         DB::table('roles')->insert([
```

Comment:
I think everything happening in this table will need to be done manually in the already existing DBs, correct?

Reply:
Yes, that is correct